test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] It is required that this file must be ignored. Kindly follow the links i provided above to correct this. You can check this [article](https://www.freecodecamp.org/news/gitignore-file-how-to-ignore-files-and-folders-in-git/)